js/flightlog_fields_presenter.js

```diff
@@ -1125,6 +1166,36 @@ function FlightLogFieldPresenter() {
                         default:
                             return value.toFixed(0) + 'us';
                     }
+                case 'GPS_RESCUE_VELOCITY':
+                    switch (fieldName) {
+                        case 'debug[0]': // Pitch P degrees * 100
+                        case 'debug[1]': // Pitch D degrees * 100
+                            return (value / 100).toFixed(1) + 'deg';
+                        case 'debug[2]': // velocity to home cm/s
+                        case 'debug[3]': // velocity target cm/s
+                            return (value / 100).toFixed(1) + 'm/s';
+                    }
+                case 'GPS_RESCUE_HEADING':
+                    switch (fieldName) {
+                        case 'debug[0]': // Rescue yaw rate deg/s * 10 up to +/- 90 
```

Comment:
Sonar issue: remove the trailing space at the end

Reply:
thanks